motadata.css

```diff
@@ -0,0 +1,77 @@
+.k-grid-toolbar {
```

Comment:
Don't create css at root level. Please keep it inside component.

Reply:
Ok